### PR TITLE
feat/feedback

### LINKED ISSUE #178: Feedback
**Describe the solution you'd like**
As a user I want to send feedback to the development team

5

### PR DESCRIPTION
**Pull request by zohra051**

### CODE REVIEW

File: lib/pages/Mainpage.dart [hunk 1]
```diff
@@ -119,29 +121,81 @@ class MainPageState extends State<MainPage> {
...
+                ListTile(
+                  title: Text('Feedback'),
+                  onTap: () {
+                    sendFeedback();
+                  },
+                ),
+                ListTile(
+                  title: Text('Contact'),
+                  onTap: () {
+                    // Update the state of the app.
```
Comment (PiotrFLEURY) [inline lib/pages/Mainpage.dart]: what is that ???

File: lib/pages/Mainpage.dart [hunk 2]
```diff
@@ -119,29 +121,81 @@ class MainPageState extends State<MainPage> {
...
+    );
+  }
+
+  void sendFeedback() async {
+    final _controller = TextEditingController();
+
+    Navigator.pop(context);
+    final Email email = Email(
+      body: _controller.text,
+      subject: 'Feedback Mobileone',
```
Comment (PiotrFLEURY) [inline lib/pages/Mainpage.dart]: Please use a translation for this

**Review by PiotrFLEURY — CHANGES_REQUESTED** [review]

File: lib/pages/Mainpage.dart [hunk 3]
```diff
@@ -118,30 +120,79 @@ class MainPageState extends State<MainPage> {
...
+      ),
+    );
+  }
+
+  sendFeedback() async {
+    final _controller = TextEditingController();
+    Navigator.pop(context);
+    final Email email = Email(
+      body: _controller.text,
+      subject: feedback,
```
Comment (nathanvandeperre97) [inline lib/pages/Mainpage.dart]: Why create a variable and not use getstring directly ?

**Review by nathanvandeperre97 — DISMISSED** [review]

File: lib/pages/Mainpage.dart [hunk 4]
```diff
@@ -119,29 +120,77 @@ class MainPageState extends State<MainPage> {
...
+                        height: 100,
+                      ),
+                    ],
+                  ),
+                  decoration: BoxDecoration(
+                    color: _colorsApp.buttonColor,
+                  ),
+                ),
+                ListTile(
+                  title: Text('Feedback'),
```
Comment (PiotrFLEURY) [inline lib/pages/Mainpage.dart]: please use translation

File: lib/pages/Mainpage.dart [hunk 5]
```diff
@@ -119,29 +120,77 @@ class MainPageState extends State<MainPage> {
...
+                  onTap: () {
+                    sendFeedback();
+                  },
+                ),
+                ListTile(
```
Comment (PiotrFLEURY) [inline lib/pages/Mainpage.dart]: seen with Zohra. To remove

**Review by PiotrFLEURY — CHANGES_REQUESTED** [review]

**Review by PiotrFLEURY — DISMISSED** [review]

**Review by PiotrFLEURY — APPROVED** [review]

**Pull request merged**